src/main.cpp

```diff
@@ -612,6 +614,11 @@ void print_help_message() {
   print_option("", "gnu-like (default if omitted)");
   print_option("", "vim-qflist-json");
   print_option("", "emacs-lisp");
+  print_option("--escape-errors=[WHEN]",
+               "Use escape sequences to show errors as hyperlinks.");
```

Comment:
Must fix: `--escape-errors` is a bad name. Let's call it `--diagnostic-hyperlinks` instead.

The fact that we use escape sequences is an implementation detail.